.github/FUNDING.yml

```diff
@@ -0,0 +1,7 @@
+github:
```

Comment:
This seems to be just a template file, shouldn't it be fulfilled with Mosqlimate info?

Reply:
We can just remove this